flask_pydantic/core.py

```diff
@@ -288,6 +291,50 @@ def wrapper(*args, **kwargs):
 
             return res
 
+        # register this decorator's args
+        setattr(wrapper, "_body", body)
+        setattr(wrapper, "_query", query)
+        return wrapper
+
+    return decorate
+
+
+def openapi_docs(
```

Comment:
Can we please move this method out of the `core` module. It's already larger than it should be I think. I will hopefully refactor it some day.